labels.mss

```diff
@@ -63,21 +63,21 @@
     shield-line-spacing: -2;
     shield-text-dx: 6;
   }
-  [type='hc'],[type='ambassade'],[type='capitale'] {
+  [type='embassy'],[type='capital'] {
```

Comment:
If we have one to one matching between types and icons, you can do something like

    shield-file: url('icon/place/[type].svg');

Reply:
Done